R/data-objects.R

```diff
@@ -69,11 +146,27 @@ iput <- function(
     stop("Local object or file does not exist.", call. = FALSE)
```

Comment:
This would only be valid for objects, not files, right? So I would change the error message to:
"Local object does not exist in current environment."